rest/src/main/java/org/jboss/pnc/rest/endpoint/RunningBuildRecordEndpoint.java

```diff
@@ -88,9 +70,8 @@ public RunningBuildRecordEndpoint(BuildRecordProvider buildRecordProvider) {
     @GET
     public Response getAll(@ApiParam(value = PAGE_INDEX_DESCRIPTION) @QueryParam(PAGE_INDEX_QUERY_PARAM) @DefaultValue(PAGE_INDEX_DEFAULT_VALUE) int pageIndex,
             @ApiParam(value = PAGE_SIZE_DESCRIPTION) @QueryParam(PAGE_SIZE_QUERY_PARAM) @DefaultValue(PAGE_SIZE_DEFAULT_VALUE) int pageSize,
-            @ApiParam(value = SORTING_DESCRIPTION) @QueryParam(SORTING_QUERY_PARAM) String sort,
-            @ApiParam(value = QUERY_DESCRIPTION, required = false) @QueryParam(QUERY_QUERY_PARAM) String q) {
-        return fromCollection(buildRecordProvider.getAllRunning(pageIndex, pageSize, sort, q));
```

Comment:
That resource does not come from the database so RSQL & Criteria API cannot be used. Before I removed the 'sort' and 'q' parameters they were there but didn't work so i didn't see a reason to keep them because that was misleading. I would have to parse the sort string and use reflection to implement the same functionality but i didn't see the need because I sort by 'id' everywhere so iIjust hardcoded that.